packages/application-generic/src/usecases/add-job/add-job.usecase.ts

```diff
@@ -36,6 +36,10 @@ export class AddJob {
     private addDelayJob: AddDelayJob
   ) {}
 
+  public getQueueService(): QueueService {
+    return this.queueService;
+  }
```

Comment:
you don't need to expose it this way, in the test you can do `(useCase as any).queueService`

Reply:
Changed the visibility to public readonly as I consider bad advise with hacking the type away as I mentioned in previous PRs.